fulp_modules/features/antagonists/bloodsuckers/code/clans/clan_ventrue.dm

```diff
@@ -5,19 +5,15 @@
 
 /datum/bloodsucker_clan/ventrue
 	name = CLAN_VENTRUE
-	description = "The Ventrue Clan is extremely snobby with their meals, and refuse to drink blood from people without a mind. \n\
-		You may only level yourself up to Level %MAX_LEVEL%, anything further will be ranks to spend on their Favorite Vassal through a Persuasion Rack. \n\
-		The Favorite Vassal will slowly turn more Vampiric this way, until they finally lose their last bits of Humanity."
+	description = "The Ventrue are extremely snobby with their meals, and refuse to drink blood from people without a mind. \n\
+		They may only gain three abilities, with the rest of their progress being given to (and to some extent shared with,) their favorite vassal. \n\
```

Comment:
> Levels affect more than just abilities they should still be let known they cannot rank up

In my experience playing Ventrue on the server, I've found that they actually _can_ rank up past level three contrary to what current descriptions indicate. They do so whenever they give their Favorite Vassal a new power and stop by their coffin to claim the rank. I haven't actually been able to recheck this on a GitHub/VScode build of the server though, since vassalizing another mob seems to require it to have a separate mind.